src/components/headerTransparent.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../styles/headerTransparent.css'; 
+import StickyNavBar from './stickyNavBar';
+
+const HeaderTransparent = () => {
+  return (
+    <div className="sticky-header">
+      <StickyNavBar/>
```

Comment:
I am a little bit confused here. Can we combine this component with the StickyNavBar component? I don't see other functionality here besides the div and class, unless I'm missing something.

Reply:
you're right, forgot we werent using this component anymore but have deleted it now mb